testing/people-tracking/app.py

```diff
@@ -0,0 +1,137 @@
+import logging
+from collections import deque
+from typing import Tuple, List
+import sys
+from pathlib import Path
+from time import monotonic
+
+import depthai_bind
+from depthai import misc
+
+import depthai as dai
+import cv2
+import numpy as np
+import blobconverter
+
+
+class PeopleTrackerDrawer:
```

Comment:
This class should be split, because it doesn't only draw stuff but it also handles tracking. It would be best to split tracking and drawing into two separate entities

Reply:
Agree